newswires/app/conf/SearchBuckets.scala

```diff
@@ -0,0 +1,73 @@
+package conf
+
+import db.SearchParams
+
+object Subjects {
+  val sportsRelated = List(
+    "medtop:15000000",
+    "iptccat:SFF",
+    "iptccat:SSO",
+    "iptccat:STR",
+    "iptccat:SRR",
+    "iptccat:SRD",
+    "iptccat:SRN",
+    "iptccat:SDP",
+    "iptccat:SDR",
+    "iptccat:SSS",
+    "iptccat:SRS",
+    "iptccat:SRZ",
+    "iptccat:SDS",
+    "iptccat:SDQ",
+    "iptccat:SCR",
+    "iptccat:SST",
+    "iptccat:RSR",
```

Comment:
I suspect that `iptccat:RDR` and `iptccat:RRR` are for horse racing (I did a couple of searches and that's all it returned) but not sure how to confirm their meaning..?